extension/persistence/eclipselink/src/main/java/io/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreSessionImpl.java

```diff
@@ -82,6 +84,10 @@ public class PolarisEclipseLinkMetaStoreSessionImpl implements PolarisMetaStoreS
   private static final Logger LOG =
       LoggerFactory.getLogger(PolarisEclipseLinkMetaStoreSessionImpl.class);
 
+  // Cache to hold the EntityManagerFactory for each realm. Each realm needs a separate
+  // EntityManagerFactory since it connects to different databases
+  private static ConcurrentHashMap<String, EntityManagerFactory> realmFactories =
```

Comment:
-> `static final` as we don't want any object of this class to reassign a new `realmFactories`